site/pages/react/authentication-methods.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: Authenticate Users
+title: Authentication Methods
```

Comment:
Looking and playing with the site. I think it might actually make sense to break it in pages and subpages like this:

Authentication (Not Authentication Methods)
|_ Getting Started (new page)
|_ Using UI Components (new page)
|_ Using React Hooks (new page)

The top level Authentication can not have any content like "Using smart accounts"

And then everything before `## Using our UI Components` can go into the Getting Started Page.